luxonis_ml/nn_archive/config_building_blocks/base_models/input.py

```diff
@@ -66,39 +68,33 @@ class Input(BaseModelExtraForbid):
     input_type: InputType = Field(description="Type of input data (e.g., 'image').")
     shape: List[int] = Field(
         min_length=1,
-        max_length=5,
         description="Shape of the input data as a list of integers (e.g. [H,W], [H,W,C], [N,H,W,C], ...).",
     )
-    layout: str = Field(
-        description="Lettercode interpretation of the input layout (e.g., 'NCHW').",
-        default="NCHW",
+    layout: List[str] = Field(
```

Comment:
It probably shouldn't be a breaking change; I also think this might be better to be optional.

Reply:
Can we test to make sure?

Reply:
It wouldn't be breaking on itself, but it might cause issues if we count on the default being "NCHW" somewhere on the hub side. The `layout` itself was fairly recent addition so it should be okay, not sure who to ask though.  I'll merge this without the `Optional` for now so it's not blocking `modelconverter` and we can change it to `Optional` later when we know it's not breaking anything.